lapack/testlapack/dlansb.go

```diff
@@ -0,0 +1,133 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"golang.org/x/exp/rand"
+	"gonum.org/v1/gonum/blas"
```

Comment:
Blank line between us and them.

Reply:
When the machine does 99% of what you need ...

Done.